README.md

```diff
@@ -11,731 +11,18 @@ STYLE CONVENTION USED
-# number density (with has_intrinsic_alignment=False)
-lens1 = ccl.WeakLensingTracer(cosmo, dndz=(z_n, n))
-lens2 = ccl.WeakLensingTracer(cosmo, dndz=(z_n, n))
-
-# Calculate the angular cross-spectrum of the two tracers as a function of ell
-ell = np.arange(2, 10)
-cls = ccl.angular_cl(cosmo, lens1, lens2, ell)
-print cls
-````
+Installation instructions can be found in `INSTALL.md` in this directory. Documentation for`CCL` can be found in `DOC.md` in this directory, in our [wiki](https://github.com/LSSTDESC/CCL/wiki), in the `CCL` [readthedocs page](https://readthedocs.org/projects/ccl/), and in the doxygen docs contained in the `doc` folder within the repository. There are also multiple examples in C, python and jupyter notebooks available in the `examples` folder.
 
+For known installation issues and further information on how CCL was benchmarked during development, see our [wiki](https://github.com/LSSTDESC/CCL/wiki).
 
 # License, Credits, Feedback etc
-The `CCL` is still under development and should be considered research in progress. You are welcome to re-use the code, which is open source and available under terms consistent with [BSD 3-Clause](https://opensource.org/licenses/BSD-3-Clause) licensing. If you make use of any of the ideas or software in this package in your own research, please cite them as "(LSST DESC, in preparation)" and provide a link to this repository: https://github.com/LSSTDESC/CCL. For free use of the `CLASS` library, the `CLASS` developers require that the `CLASS` paper be cited: CLASS II: Approximation schemes, D. Blas, J. Lesgourgues, T. Tram, arXiv:1104.2933, JCAP 1107 (2011) 034. The `CLASS` repository can be found in http://class-code.net. If you have comments, questions, or feedback, please [write us an issue](https://github.com/LSSTDESC/CCL/issues). Finally, CCL uses code from the [FFTLog](http://casa.colorado.edu/~ajsh/FFTLog/) package.  We have obtained permission from the FFTLog author to include modified versions of his source code.
+The `CCL` is still under development and should be considered research in progress. You are welcome to re-use the code, which is open source and available under terms consistent with [BSD 3-Clause](https://opensource.org/licenses/BSD-3-Clause) licensing. If you make use of any of the ideas or software in this package in your own research, please cite them as "(LSST DESC, in preparation)" and provide a link to this repository: https://github.com/LSSTDESC/CCL. For free use of the `CLASS` library, the `CLASS` developers require that the `CLASS` paper be cited: CLASS II: Approximation schemes, D. Blas, J. Lesgourgues, T. Tram, arXiv:1104.2933, JCAP 1107 (2011) 034. The `CLASS` repository can be found in http://class-code.net. Finally, CCL uses code from the [FFTLog](http://casa.colorado.edu/~ajsh/FFTLog/) package.  We have obtained permission from the FFTLog author to include modified versions of his source code.
```

Comment:
Sorry, I missed this when I reviewed earlier: I think it's a bit inconsistent to say CCL is publicly released and also it's still research in progress.  Of course we will continue to develop past v1 as well, but it seems valuable to indicate something about the actual usability, validation, stability, etc.  Something like "This code has been released by DESC, although it is still under active development.  When it reaches v1 status it will be accompanied by a journal paper that describes the development and validation of CCL."?